clients/rust/src/generated/types/decompressable_state.rs

```diff
@@ -0,0 +1,16 @@
+//! This code was AUTOGENERATED using the kinobi library.
+//! Please DO NOT EDIT THIS FILE, instead use visitors
+//! to add features, then rerun kinobi to update it.
+//!
+//! [https://github.com/metaplex-foundation/kinobi]
+//!
+
+use borsh::BorshDeserialize;
+use borsh::BorshSerialize;
+
+#[derive(BorshSerialize, BorshDeserialize, Clone, Debug, Eq, PartialEq, PartialOrd, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum DecompressableState {
```

Comment:
Maybe its too late for our program overall because the instruction is even called `set_decompressable_state`, but I think this should be spelled "decompressible" with an "i" not an "a".

Reply:
Let's fix it. We can create a new instruction on the program with the correct name, so the "current" one just calls this new one. On the Rust client, we remove the misspelled one so only the "correct" one can be used.

I will create a separate PR for that.